comment.go

```diff
@@ -7,8 +7,10 @@ import (
 )
 
 const (
-	ignoreComment  = "//exhaustive:ignore"
-	enforceComment = "//exhaustive:enforce"
+	ignoreComment                = "//exhaustive:ignore"
+	enforceComment               = "//exhaustive:enforce"
+	defaultRequireIgnoreComment  = "//exhaustive:default-require-ignore"
+	defaultRequireEnforceComment = "//exhaustive:default-require-enforce"
```

Comment:
I agree that formatting is preferable. I wrote it in this reversed way because "//exhaustive:ignore-default-require(d)" is a prefix of "//exhaustive:ignore", so that induces some significance to the order in which we try to parse directives. (I believe a careful ordering of `if` statements, like how you suggested for the precedence issue above, should cover it, longest-first). I'm happy to make that change but absent feedback went with no-common-prefix approach.

So if inducing that significance sounds good, let me know and I'll happily make that change. Thanks!

Reply:
I forgot to add: I suppose it is technically backwards-incompatible, because if by some extremely-strange-chance a user already had "//exhaustive:ignore-..." it will no longer do the old behavior. I am OK with that but I'm not the maintainer :), so I should raise that too. (More realistically, this is why I didn't consider renaming the existing "//exhaustive:ignore" directives, so users won't see any change-of-behavior there.)

Reply:
Thanks for mentioning and implementing the longest-first ordering. I missed to consider it. 

> I forgot to add: I suppose it is technically backwards-incompatible, because if by some extremely-strange-chance a user already had "//exhaustive:ignore-..." it will no longer do the old behavior. I am OK with that but I'm not the maintainer :), so I should raise that too.

Thanks for mentioning this, too. It is unfortunate that comments of this form, if they exist, will break. I wish that the current code in master was more strict about the requirements here. That is, I wish that the code in master performed the equivalent of a `^//exhaustive:ignore( \t|$)` check. Then the following comments would be valid, as desired:

('•' is used to visually represent space below)

```go
//exhaustive:ignore
//exhaustive:ignore•
//exhaustive:ignore•more•text
```

and the following would be invalid, as desired:

```go
//exhaustive:ignore-more-text
```

The package doc in master, too, doesn't go into this level of detail. It just says, "a comment that begins with". 

Overall, I'm ok with this being a breaking change, too. I will mention it in the release notes. I'll also create a separate issue to consider updating the code in master to have the level of strictness described above.